### ISSUE TITLE
Go mod has non-....v2 module path

### ISSUE THREAD

**Issue by ianaz** [gavv/httpexpect#60]
Hi,

I'm using go mod for dependencies and when I retrieve the dependency with

`go get gopkg.in/gavv/httpexpect.v2`

I get the following error
```
go: downloading gopkg.in/gavv/httpexpect.v2 v2.0.1
go get gopkg.in/gavv/httpexpect.v2: go.mod has non-....v2 module path "github.com/gavv/httpexpect" at revision v2.0.1
```

This can easily be reproduced in a fresh folder. Just create a new `go mod init anyname`

Then call
`go get gopkg.in/gavv/httpexpect.v2`

**OR**

create a go.mod file with the following content 
```
module git.blockfactory.com/certification

go 1.13

require (
 gopkg.in/gavv/httpexpect.v2 v2.0.1
)
```

and call `go get`

**Comment by ianaz** [comment]
Sorry, I noticed the .v2 path is for development only.
With `go get github.com/gavv/httpexpect everything works :)`

**Comment by gavv** [comment]
Hi,

gopkg and go.mod are not compatible. gopkg is an ancient way to version go packages appeared before go.mod. httpexpect supports both.

If you're using go.mod, you should use `github.com/gavv/httpexpect/v2`. Otherwise, you should use `gopkg.in/gavv/httpexpect.v2`.

It appeared that the README was outdated. I've updated it: https://github.com/gavv/httpexpect#versions.

It also appeared that our go.mod was missing the `/v2` component. I've fixed it: f1807e85af0b542c4f114936650092eb0ac1bf6a and tagged a new release `v2.0.2`.

Thanks for reporting.
